admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/controller/BiometricTypeControllerTest.java

```diff
@@ -122,7 +122,7 @@ public void t5createBiometricTypeFailTest() throws Exception {
 	@Test
 	@WithUserDetails("global-admin")
 	public void t6getAllBiometricTypesFailTest() throws Exception {
-		MasterDataTest.checkResponse(mockMvc.perform(MockMvcRequestBuilders.get("/biometrictypes")).andReturn(),"KER-MSD-006");
```

Comment:
why for failed test cases we are adding null or modify the test name

Reply:
Modified the name sir.